requirements_all.txt

```diff
@@ -933,6 +933,9 @@ pyzabbix==0.7.4
 # homeassistant.components.sensor.qnap
 qnapstats==0.2.4
 
+# homeassistant.components.sensor.qnap
+qqbot==2.3.7
```

Comment:
You should not update requirements_all.txt manually, but run script/gen_requirements_all.py
More info here https://home-assistant.io/developers/development_testing/